presto-plugin-toolkit/src/test/resources/table.json

```diff
@@ -1,4 +1,18 @@
 {
+  "schemas": [
+    {
+      "user": "user_with_access_to_bobschema",
+      "schema": "bobschema",
+      "owner": false,
+      "allow": "read-only"
+    },
+    {
+      "user": "user_without_access_to_bobschema",
+      "schema": "bobschema",
+      "owner": false,
+      "allow": "none"
+    }
+  ],
```

Comment:
Can you please add a test where user has access to insert to all tables (schema pattern `.*`), but is allowed to `all`, `read-only` and `none` some schemas?

Reply:
I add ` user_with_none_access_to_bobschema ` 
and ` user_with_read_only_access_to_bobschema `
Both user has ` INSERT ` privileges in table rule for all schema (` schema: .* `)
meanwhile, I added schemaRule for both user
` user_with_none_access_to_bobschema ` with ` allow: none `
and
` user_with_read_only_access_to_bobschema ` with ` allow: read-only `
in test, both user get deny when ` INSERT ` into ` boboschema.bobtable `